spinalcordtoolbox/reports/qc.py

```diff
@@ -618,6 +740,7 @@ def generate_qc(fname_in1, fname_in2=None, fname_seg=None, angle_line=None, args
     each slice, for slice that don't have an angle to display, a nan is expected. To be used for assessing cord orientation.
     :param args: args from parent function
     :param path_qc: str: Path to save QC report
+    :param fps: float: Number of frames per second for output gif images
```

Comment:
Right now, `fps` is only valid for moco, right? It might be good to be explicit about this in the parameter descriptions.

Maybe even run a check to ensure that `fps` is only passed for `process` == `['sci_dmri_moco', 'sct_fmri_moco']`.